purchase_buyer_contact/README.rst

```diff
@@ -0,0 +1,19 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Purchase Buyer Contact
+=========================
+
+This modules allow you to define the name, phone and email of the person in charge of purchase in your company.
+You can define this information against the res.company object.
+All res.partner created will then have the same info by default, but you can modify it.
+You can also define against the res.partner the code the supplier uses for you.
```

Comment:
Do you mean the code that a supplier uses to represent your company? This does not seem to be related to the concept of Purchase Buyer Contact. Can you please move it to another module? I'm also not convinced that this is the right technical solution to define how to enter the name that a supplier uses for your company.